src/messenger-shortcuts.js

```diff
@@ -148,7 +149,8 @@ function toggleInfo() {
 }
 
 function getSearchBar() {
-  return getByAttr('input', 'type', 'text');
+  return document.querySelector('input[type="text"][placeholder="Search Messenger"]');
```

Comment:
Any reason for the extra condition? Are there cases where it doesn't work without it? (I'm worried that they'll quietly change the "Search Messenger" text at some point.)